dist/profile/manifests/kubernetes/resources/datadog.pp

```diff
@@ -0,0 +1,62 @@
+# Deploy datadog resources on kubernetes cluster
+class profile::kubernetes::resources::datadog (
```

Comment:
This whole class might make better sense as a defined resource, see `datadog_check.pp` as an example. Then the usage could be parameterized to point to secrets, and resource names in the templates directory